platforms/enterprise/enterprise/src/main/java/org/gradle/internal/enterprise/impl/DefaultGradleEnterprisePluginAdapter.java

```diff
@@ -112,6 +113,11 @@ public void buildFinished(@Nullable Throwable buildFailure, List<org.gradle.inte
     }
 
     private void createPluginService() {
+        String injectedDevelocityUrl = config.getDevelocityUrl();
+        if (injectedDevelocityUrl != null) {
+            // We might want to stop setting this system property if the Develocity plugin is new enough to read the URL from the configuration directly.
+            System.setProperty("com.gradle.scan.server", injectedDevelocityUrl);
```

Comment:
Do we need to check if this is already set and do something else (warn or fail)?